inst/schema/ProjectState.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "file": {
+      "type": "object",
+      "properties": {
+        "path": { "type": "string" },
```

Comment:
Will these be empty string if file was an optional one which wasn't used e.g. ANC?

Reply:
Oh yeah good catch. So in the model submit `programme` and `anc` are just not required properties i.e.

```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "properties": {
    "pjnz": { "$ref": "SessionFile.schema.json" },
    "shape": { "$ref": "SessionFile.schema.json" },
    "population": { "$ref": "SessionFile.schema.json" },
    "survey": { "$ref": "SessionFile.schema.json" },
    "programme": { "$ref": "SessionFile.schema.json" },
    "anc": { "$ref": "SessionFile.schema.json" }
  },
  "additionalProperties": false,
  "required": [
    "pjnz",
    "shape",
    "population",
    "survey"
  ]
}
```

could go for the same approach here for consistency

Reply:
Fixed this and added a test too. Will address notes comment in a separate PR